kernel/src/misc/wm/wm.c

```diff
@@ -554,15 +582,18 @@ void wm_mouse_callback(mouse_t raw_curr) {
 }
 
 void wm_kbd_callback(kbd_event_t event) {
+    wm_event_t kbd_ev;
```

Comment:
Rename to `event` or `kbd_event` and assign directly its type etc... here. Reuse within the loop.